zsh/.zshrc

```diff
@@ -257,6 +272,14 @@ if [[ -z "$LS_COLORS" ]]; then
   export LS_COLORS='di=1;34;40:ln=35;40:so=32;40:pi=33;40:ex=31;40:bd=34;46:cd=34;43:su=0;41:sg=0;46:tw=0;42:ow=0;43:'
 fi
 
+# Should this be indie this be inside load-our-ssh-keys()?
+if ! can_haz ssh-askpass; then
+  echo
+  echo "For the quickstart to load your ssh keys, enable the following zqs setting: "
+  echo $(tput setaf 2) "zqs enable-ssh-askpass-require true" $(tput setaf 7)
```

Comment:
I was testing this here to see if we can make it stand out on the terminal. I can remove if you don't think it makes sense.